src/app/detail/[task_id]/[user_type]/actions.ts

```diff
@@ -4,7 +4,7 @@ import { advancedFeatureFlag, apiUrl } from '@/config'
 import { CreateAttachmentRequest } from '@/types/dto/attachments.dto'
 import { CreateComment } from '@/types/dto/comment.dto'
 import { UpdateTaskRequest } from '@/types/dto/tasks.dto'
-import { revalidatePath, revalidateTag } from 'next/cache'
+import { revalidateTag } from 'next/cache'
```

Comment:
remind me why we removed revalidatePath?

Reply:
We have removed `revalidatePath` from here because from now onwards we are depending on the `RealTime` HOC for updating page after mutations by updating the `tasks` state in the redux store. For the real time functionality, after any database events we could have refetched the data, but this approach would take time to fetch and it is our requirement to not have any loading states in the app while doing activities like dragging/dropping and task creation. For that reason, when we started with real time functionality, we decided to remove revalidation and instead update the state in the redux.